SimTKmath/Optimizers/src/IpOpt/IpLapackSolverInterface.cpp

```diff
@@ -146,6 +146,7 @@ double *afact;
     delete [] a;
     delete [] irn_;
     delete [] jcn_;
+    delete [] ipiv_; 
```

Comment:
Should set `ipiv_ = nullptr;` here -- the other three are reassigned but ipiv is not.